pkgs/desktops/gnome-3/misc/gitg/src.nix

```diff
@@ -1,10 +1,12 @@
 # Autogenerated by maintainers/scripts/gnome.sh update
 
-fetchurl: {
-  name = "gitg-3.24.0";
+fetchurl: rec {
+  major = "3.26";
+  minor = "0";
+  name = "gitg-${major}.${minor}";
 
   src = fetchurl {
-    url = mirror://gnome/sources/gitg/3.24/gitg-3.24.0.tar.xz;
-    sha256 = "3e4ec4a8ae83bc7ced8c7610927ade70e37daa5e8beeb4f357a6ea30b4cc951e";
+    url = "mirror://gnome/sources/gitg/${major}/${name}.tar.xz";
```

Comment:
The attributes will be erased in the next gnome update. You can update the file the same way using `maintainers/scripts/gnome.sh update gitg 3.26`.